lib/pages/forms_pages/components/add_photo_button_widget.dart

```diff
@@ -14,8 +14,9 @@ class AddPhotoButton extends StatefulWidget {
   final bool isEditing;
   final bool photoRequired;
   final Function _atLeastOnePhotoAttached;
+  final bool isAdultReport;
 
-  AddPhotoButton(this.isEditing, this.photoRequired, this._atLeastOnePhotoAttached);
+  AddPhotoButton(this.isEditing, this.photoRequired, this._atLeastOnePhotoAttached, this.isAdultReport);
```

Comment:
I think it's better to make the AddPhotoButton class agnostic of the type of report. I propose using a string variable like `this.subtitle` or similar, and only show the text if that variable is not null or empty.

Reply:
you're right, if modified it. I'll do the same change for the other PR too